ddtrace/ext/priority.go

```diff
@@ -9,6 +9,11 @@ package ext
 // In a distributed context, it should be set before any context propagation (fork, RPC calls) to be effective.
 
 const (
+	// PrioritySamplingRuleReject informs the backend that a trace should be rejected and not stored as specified by
+	// a sampling rule.
+	// This should be used by the rules sampler based on the rules configured by the user
+	PrioritySamplingRuleReject = -3
```

Comment:
```suggestion
	PriorityRuleSamplerReject = -3
```
I think this will make it much clearer that this is here to clarify that the "rule sampler" made this choice. WDYT?